components/commons/footer/Footer.tsx

```diff
@@ -0,0 +1,6 @@
+import { FooterProps } from "./Footer.types";
+import "./styles/Footer.module.scss";
+
+export default function Footer({}: FooterProps) {
```

Comment:
`FC` stands for FunctionalComponent Type, which is a type provided by React for Functional Components

```suggestion
const Footer: FC<Props> = ({}) => {
```